count_vowels.py

```diff
@@ -0,0 +1,58 @@
+"""
+Hometask 4 - Count vowels.
+
+# The English language has five vowels: A, E, I, O, and U
+# Please count each vowel occurrence in text bellow
+# (total sum of lower and capital cases)
+# and write output as table something like this
+#  -----------------
+#  | vowel | count |
+#  -----------------
+#  |   a   |  11   |
+#  |   e   |  23   |
+#    ...
+#  -----------------
+# then modify text where each vowel replaced with
+# A->À;  a->à ; E-> É ; e->é; I->Í , i->í ; O->Ó ; o->ó; U->Ú; u->ú
+# ex. "Í wàndéréd lónély..." and print it.
+"""
+
+poem_text = """I wandered lonely as a cloud
+That floats on high o'er vales and hills,
+When all at once I saw a crowd,
+A host, of golden daffodils;
+Beside the lake, beneath the trees,
+Fluttering and dancing in the breeze.
+Continuous as the stars that shine
+And twinkle on the Milky Way,
+They stretched in never-ending line
+Along the margin of a bay:
+Ten thousand saw I at a glance,
+Tossing their heads in sprightly dance."""
+
+"""
+We count the vowels in the text and put them in the dictionary.
+And after that we display a table with the counted vowels.
+"""
+
+vowels = {'A': poem_text.count('a'), 'E': poem_text.count('e'),
```

Comment:
Pay attention that you need to get count of  lower and capital letters ( count 'A' + count 'a') or do it somehow in more smart way. Now solution gives wrong result/ It should be fixed